src/ModuleTestLite.cs

```diff
@@ -341,7 +341,12 @@ private void triggerFailure(int type)
 					type = (int)failureTypes.PERFLOSS;
 				}
 			}
-			failure_du += failureData[type];
+            // No failure data until the craft has launched
```

Comment:
Indentation mismatch.  TestLite uses hard tabs (and these are 8 columns wide).
Also, I'd prefer C-style /* winged comments */ for this kind of describe-the-verb comment (I'm aware I haven't quite been consistent about this elsewhere).